pallets/process-validation/src/lib.rs

```diff
@@ -29,11 +30,29 @@ impl Default for ProcessStatus {
     }
 }
 
-#[derive(Encode, Decode, Default, Clone, PartialEq)]
+#[derive(Encode, Decode, Clone, PartialEq)]
+#[cfg_attr(feature = "std", derive(Debug))]
+pub struct Version {
+    version: i32, // TODO: sort this type, should be included from trait
+}
+// TODO remove once type has been soprted <version>
+impl EncodeLike<i32> for Version {}
+
+
+#[derive(Encode, Decode, Clone, PartialEq)]
 #[cfg_attr(feature = "std", derive(Debug))]
 pub struct Process {
     status: ProcessStatus,
-    restrictions: Vec<Restriction>
+    restrictions: Vec<Restriction>,
+}
+
+impl Default for Process {
+    fn default() -> Process {
+        Process {
+            status: ProcessStatus::Disabled,
+            restrictions: vec![{ Restriction::None }],
+        }
+    }
```

Comment:
Why re-implment the `Default` trait? This was being derived by 

```
#[derive(Encode, Decode, Default, Clone, PartialEq)]
```

Reply:
mmm.... might need to read a bit more on Default trait and what it does, noted!